system-int-test/test/cmr/system_int_test/ingest/translation_test.clj

```diff
@@ -220,32 +220,3 @@
                                            :collection :dif dif9-xml :umm-json
                                            {:query-params {"skip_umm_validation" "true"}})]
       (is (= 200 status)))))
-
-(comment
```

Comment:
I think this is one of those comment blocks that is really useful to have around for debugging UMM issues. Check with Leo and Siwei before removing this.

Reply:
I have not used this comment block code personally.